source/_integrations/recorder.markdown

```diff
@@ -17,6 +17,12 @@ This integration constantly saves data. If you use the default configuration, th
 
 </div>
 
+<div class='note'>
+
+This integration does not affect the data, that is calculated and stored in the database by the [`statistics`](/integrations/statistics/) integration.
```

Comment:
This is not true, if the data isn't recorded, we cannot generate statistics either.

Reply:
Ok, thanks for this clarification. Then, the recorder integration is related to the statistics (a very importan relation). That should then be mentioned in the docs. What about such a text?

> This integration does affect the data, that is calculated and stored in the database by the statistics. The statistics  integration can only generate statistics for data that is recorded.

IMHO there must be a time, after the statistics where generated, that some parts of the states and the events could be cleaned (purged) without affecting the statistics. But how is that timing? What is needed to calculate/recalculate the statistics?

Reply:
statics integration is something different?

I think we (both) are mixing up long-term statistics feature vs statistics integration. Anyways, if there are specifics for the statistics integration, it needs to be documented on that page specifically (and not somewhere else).

Reply:
Ah, I was perhaps irritated by this page:
https://data.home-assistant.io/docs/statistics/

It describes the `long-term statistics feature` and not the statistics integration that is documented [here](https://www.home-assistant.io/integrations/statistics). (Perhaps the name could be not the same)

So something like this?

```suggestion
This integration does affect the data, that is calculated and stored in the database by the [`long-term statistics`](https://data.home-assistant.io/docs/statistics) feature. The long-term statistics feature will only generate statistics for data that is recorded.
```

It can also be written in the docs of the  `long-term statistics feature`, but I would suggest to support a normal user with the information, that purging/excluding sensor data with the recorder can have effects on the long-term statistics.